sabnzbd/interface.py

```diff
@@ -446,6 +450,28 @@ def shutdown(self, **kwargs):
         sabnzbd.shutdown_program()
         return T("SABnzbd shutdown finished")
 
+    @secured_expose(check_configlock=True)
+    def backup(self, **kwargs):
+        adminpath = cfg.admin_dir.get_path()
+        logging.debug("Backing up %s + %s", adminpath, config.CFG.filename)
+        adminfiles = [f for f in os.listdir(adminpath) if os.path.isfile(os.path.join(adminpath, f))]
```

Comment:
Why use a wildcard `os.listdir`? Can't we just create a new constant in `sabnzbd.constants` that includes all the files we want to backup? Since in `sabnzbd.constants` we have constants for each of the filenames.
So we don't include any random files.

Reply:
I didn't know what it might contain. Is this the complete list?
```
ADMIN_FILES = (
    BYTES_FILE_NAME,
    QUEUE_FILE_NAME,
    POSTPROC_QUEUE_FILE_NAME,
    RSS_FILE_NAME,
    SCAN_FILE_NAME,
    RATING_FILE_NAME,
    DB_HISTORY_NAME,
)
```